tests/core/CoreTest.py

```diff
@@ -120,6 +120,13 @@ def generate_tasks(self):
         return (((i,), {}) for i in range(tasks_count))
 
 
+class DependentOnManyZeroTaskBearsTestBear(TestBearBase):
```

Comment:
`DependentOnManyZeroTaskBearsTestBear` -> `DependentOnMultipleZeroTaskBearsTestBear`

Reply:
Does it matter? :D

Reply:
Just trying to be more accurate with words. :smiley:

Reply:
Kk ;)